frontend/src/DetailsBox.js

```diff
@@ -297,4 +297,5 @@ class DetailsBox extends Component {
   }
 }
 
-export default DetailsBox
+export default React.forwardRef((props, ref) => 
+  <DetailsBox innerRef={ref} {...props}/>);
```

Comment:
I think you'll have to rewrite DetailsBox as a functional component, since there shouldn't be any passing of refs through props at all. Ideally we'd want something like forwardRef((props, ref) => { functional component code here }